language/vm/vm-runtime/src/interpreter.rs

```diff
@@ -752,6 +753,9 @@ impl<'txn> Interpreter<'txn> {
         let account_module = runtime.get_loaded_module(&ACCOUNT_MODULE, context)?;
         let account_resource = self.operand_stack.pop_as::<Struct>()?;
         let address = self.operand_stack.pop_as::<AccountAddress>()?;
+        if address == core_code_address() {
```

Comment:
Whether we keep this line of check remains debatable. Technically we've already checked it in the LibraAccount module.

Reply:
I think whether the check should be here or in the `LibraAccount` module depends on whether we think the "reserved address" should be a Libra layer feature (do it in `LibraAccount`) or a Move layer feature (do it here). I can see arguments for both. However, I think we should make a decision on this and keep the check in only once place.